src/screens/OrgPost/OrgPost.tsx

```diff
@@ -0,0 +1,501 @@
+import type { ChangeEvent } from 'react';
+import React, { useState, useEffect } from 'react';
+import { Search } from '@mui/icons-material';
```

Comment:
This file must not be included in this PR. It is out of scope

Reply:
Can you please suggest how to remove this file from the PR?. I tried but could not do it.